labelbox/schema/project.py

```diff
@@ -121,6 +121,35 @@ def members(self) -> PaginatedCollection:
                                    {id_param: str(self.uid)},
                                    ["project", "members"], ProjectMember)
 
+    def create_project_resource_tags(self, resource_tag_ids: List[str]) -> List[str]:
+        """ Creates project resource tags
+
+        Args:
+            resource_tag_ids
+        Returns:
+            a list of ResourceTag ids that was created.
+        """
+        project_id_param = "projectId"
+        tag_ids_param = "resourceTagIds"
+
+        query_str = """mutation AttatchProjectResourceTags($%s: ID!, $%s: [String!]) {
```

Comment:
queries and mutations in the SDK should end with PyApi. So this should be `mutation AttatchProjectResourceTagsPyApi(...`